src/rpc/masternode.cpp

```diff
@@ -445,15 +445,15 @@ UniValue masternode_winners(const JSONRPCRequest& request)
     int nLast = 10;
     std::string strFilter = "";
 
-    if (request.params.size() >= 2) {
+    if (!request.params[1].isNull()) {
         nLast = atoi(request.params[1].get_str());
     }
 
-    if (request.params.size() == 3) {
+    if (!request.params[2].isNull()) {
         strFilter = request.params[2].get_str();
     }
 
-    if (request.params.size() > 3)
+    if (!request.params[3].isNull())
```

Comment:
This one should stay as it was. Usually RPC commands have checks for `request.fHelp" and `request.params.size()` at the top to ensure that the correct number of required/optional args are given. In this case, one could pass a json null object as third parameter and still pass unlimited parameters after it. Doesn't really make sense and wouldn't cause harm, but still...that's not consistent with other RPC commands.

Reply:
Actually, the RPC help should be fixed...I did this in https://github.com/dashpay/dash/commit/9b62a297bc48d0b45354932ad1133be5da10bc8b